cmds/tcz/tcz.go

```diff
@@ -72,9 +72,10 @@ func clonetree(tree string) error {
 	l.Printf("Clone tree %v", tree)
 	lt := len(tree)
```

Comment:
lt is the length of the tree. I want to redo this a bit, actually, but that's for a later PR. When I wrote it originally I was not real up on the filepath package.

The idea is that you are taking a tree from a tinycore package rooted at, e.g., /tcloop/this/that/other and creating a tree of symlinks in /